include/cuco/detail/hash_functions/utils.cuh

```diff
@@ -16,6 +16,13 @@
 
 #pragma once
 
+#include <cub/cub.cuh>
+
+#include <cuda_runtime.h>
+
```

Comment:
```suggestion
```
nit: this is only needed when building with host compilers like gcc, `.cuh` files are built with `nvcc` by default thus this is redundant

Reply:
Makes sense, thanks!